mobile/src/navigations/root/root.tsx

```diff
@@ -26,27 +34,42 @@ const Root: React.FC = () => {
     const { isProfileComplete, role } = userData ?? {};
     // prettier-ignore
     return (
-      <RootStack.Navigator screenOptions={screenOptions}>
-          {isSignedIn ? (
+        <RootStack.Navigator screenOptions={screenOptions}>
+            {isSignedIn ? (
+                <RootStack.Screen
+                    name={RootScreenName.MAIN_ROOT_ROUTE}
+                    component={
+                        role === UserRole.TALENT
+                            ? TalentBottomTabNavigator
+                            : EmployerBottomTabNavigator
+                    }
+                />
+            ) : (isProfileComplete ? (
+                <RootStack.Screen
+                    name={RootScreenName.ONBOARDING_ROOT_ROUTE}
+                    // TODO: create EmployerOnboardingNavigator for role == 'employer'
+                    component={TalentOnboardingNavigator}
+                />
+            ) : (
+                <RootStack.Screen
+                    name={RootScreenName.AUTH_ROOT_ROUTE}
+                    component={AuthNavigator}
+                />
+            ))}
               <RootStack.Screen
-                  name={RootScreenName.MAIN_ROOT_ROUTE}
-                  component={role === UserRole.TALENT ? TalentBottomTabNavigator : EmployerBottomTabNavigator}
-              />
-          ) : (isProfileComplete ? (
+                name={TalentOnboardingScreenName.PROFILE}
+                component={Profile}
+            />
+                <RootStack.Screen
+                    name={TalentOnboardingScreenName.BSA_BADGES}
+                    component={BsaBadges}
+                />
               <RootStack.Screen
-                  name={RootScreenName.ONBOARDING_ROOT_ROUTE}
-                  // TODO: create EmployerOnboardingNavigator for role == 'employer'
-                  component={TalentOnboardingNavigator}
-              />
-          ) : (
-              <RootStack.Screen
-                  name={RootScreenName.AUTH_ROOT_ROUTE}
-                  component={AuthNavigator}
-              />
-          ))}
-
-      </RootStack.Navigator>
-  );
+                name={TalentOnboardingScreenName.SKILLS_AND_PROJECTS}
+                component={SkillsAndProjects}
+            />
+        </RootStack.Navigator>
+    );
```

Comment:
Why did you change our root navigation?

Reply:
I had add RootStack.Screens for navigations, but git shows than I have been deleted and add the same code as before, if there are a better way to add screens tell me please, I'll do it)
Or It's should be talanted stack for navigation?

Reply:
We have already talent onboarding navigation and in root we conditionally render it. And in that file we have all screens names and screens params etc.

Reply:
fix